packages/react-components/src/components/styles/RTESendBox.styles.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Theme, mergeStyles } from '@fluentui/react';
+
+/**
+ * @private
+ */
+export const defaultSendBoxInactiveBorderThicknessREM = 0.0625;
+/**
+ * @private
+ */
+export const defaultSendBoxActiveBorderThicknessREM = 0.125;
+
+/**
+ * @private
+ */
+export const sendBoxStyle = ({
```

Comment:
It looks very similar to `borderAndBoxShadowStyle` in `SendBox.styles.ts` . Can we reuse the same func in both places?

Reply:
I had thought this was likley going to change and put this in more as skeleton for future styles.

You're right though this is basically the same. I will just wrap the other with mergeStyles for now and it can changes later if needed.

Reply:
It looks like `borderAndBoxShadowStyle` is used in mergeStyles in SendBox too. Maybe we can just use `mergeStyles` in `borderAndBoxShadowStyle`?

Reply:
Oh good catch!

Reply:
Updated to reference Send box style and refactored sendbox to use style with mergestyles already applied.